docs/File-Formats.md

```diff
@@ -543,7 +543,7 @@ The following extra annotation columns are also important for making sure mutati
 
 *Adding your mutation annotation columns to the complete MAF rows* can also be done. In this way, the portal will parse and store the MAF fields as well. For example, mutation data that you find on cBioPortal.org comes from MAF files that have been further enriched with information from [mutationassessor.org](http://mutationassessor.org/), which leads to a 'Mutation Assessor” column in the [mutation table](http://www.cbioportal.org/index.do?cancer_study_list=acc_tcga&cancer_study_id=acc_tcga&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=acc_tcga_mutations&Z_SCORE_THRESHOLD=2.0&RPPA_SCORE_THRESHOLD=2.0&data_priority=0&case_set_id=acc_tcga_sequenced&case_ids=&patient_case_select=sample&gene_set_choice=user-defined-list&gene_list=ZFPM1&clinical_param_selection=null&tab_index=tab_visualize&Action=Submit).
 
-The MAF format recognized by the portal (excluding the annotation columns already mentioned above) has 32 columns + 4 columns with information on reference and variant allele counts in tumor and normal samples. A more detailed example MAF can be found on our [Downloads](Downloads.md#maf-example) page. Description of each column is provided below:
+The MAF format recognized by the portal (excluding the annotation columns already mentioned above) has 32 columns + 4 columns with information on reference and variant allele counts in tumor and normal samples + 4 columns with custom annotation of driver and passenger mutations. A more detailed example MAF can be found on our [Downloads](Downloads.md#maf-example) page. Description of each column is provided below:
```

Comment:
This text was a bit confusing, and now it's even more confusing I think. Maybe we can make it nice with this PR? Something like:

The MAF format recognized by the portal (excluding the annotation columns already mentioned above) has:
- 32 columns [link tcga maf format]
- 4 columns with information on reference and variant allele counts in tumor and normal samples
- 4 columns with custom annotation of driver and passenger mutations. 

A more detailed example MAF can be found on our Downloads page[The download is not working. We can link this to the maf of `study_es_0`]. Description of each column is provided below: